app/src/main/java/com/compscieddy/meetinthemiddle/HomeActivity.java

```diff
@@ -369,15 +374,30 @@ public void onChildAdded(DataSnapshot dataSnapshot, String s) {
           public void onDataChange(DataSnapshot dataSnapshot) {
             Group group = dataSnapshot.getValue(Group.class);
             mGroupsAdapter.addGroup(group);
-          }
 
+            //TODO Need to find a better place to put this
```

Comment:
Read through the basic Firebase docs before adding code in these callbacks. Very explosive 🔥 things can happen